main.c

```diff
@@ -60,11 +59,11 @@ int main(int argc, char** argv) {
         void xyzOrthographicProjection();
 
         //~~~~~ Group 17 ~~~~~//
-        int outputPoint(struct point * finalPoint, char* filename);
+        outputPoints(fileName);
 
         free(inputShape);
     }
-    free(fileName);
```

Comment:
this needs to be freed here as it is not static memory when group 1 is working with it